checks/evaluation/signed_releases_test.go

```diff
@@ -0,0 +1,108 @@
+//  Copyright 2023 OpenSSF Scorecard Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package evaluation
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+
+	"github.com/ossf/scorecard/v4/checker"
+	"github.com/ossf/scorecard/v4/clients"
+	scut "github.com/ossf/scorecard/v4/utests"
+)
+
+func TestSignedReleases(t *testing.T) {
+	tests := []struct {
+		name           string
+		releases       []clients.Release
+		expectedResult checker.CheckResult
+	}{
+		{
+			name: "Full score",
```

Comment:
optional: the test names would be clearer to me if the name included the behavior we're looking for. 
So for this test we're looking for the provenance, in the other test we're looking for signature, etc.

Reply:
I am going to skip this for now. Thanks for the feedback.